tests/caseManagement.js

```diff
@@ -169,6 +171,22 @@ define([
             chai.expect(mappedQuestions.attr("update_mode")).to.equal("edit");
         });
 
+        it("should display a dropdown of potential case management properties", function () {
+            util.loadXML("");
+            util.addQuestion("Text", "question1");
+
+            const caseManagementSection = getCaseManagementSection();
+            const casePropertySelect = caseManagementSection.find(CASE_PROPERTY_WIDGET_TYPE);
+
+            const optionValues = {};
```

Comment:
nit: It seems like it would simplify this test if an Array was used here instead of an Object. The values are never actually used, are they?

Reply:
Good suggestion: b85b8d31f9fcb283dc92bfd7e6d8faafc0ad12cf